core/src/main/resources-filtered/org/pentaho/platform/dataaccess/datasource/wizard/public/modeler.properties

```diff
@@ -142,6 +142,14 @@ validation.level.MISSING_LAT=Level must have a Latitude member property
 validation.level.MISSING_LON=Level must have a Longitude member property
 default_aggregation=Default Aggregation
 
+# Model Tree alternative text
+modeler.alternative_text.model=Model
+modeler.alternative_text.dimension=Dimension
+modeler.alternative_text.category=Category
+modeler.alternative_text.baseColumnBacked=Column
+modeler.alternative_text.availableField=Column
```

Comment:
These properties are consumed by the files in pentaho/modeler source code path : core/src/main/java/org/pentaho/agilebi/modeler/nodes/DimensionMetaData.java

see PR https://github.com/pentaho/modeler/pull/376 for such classes